### PR TITLE
Revamp docs based on review

### PR DESCRIPTION
**Pull request by nalakas**

### CODE REVIEW

File: en/docs/Learn/DesignAPI/Endpoints/EndpointSecurity/basic-auth.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
-# Basic Auth
+# Secure Endpoint with Basic Auth
 
-When you create an API using the API Publisher, you can specify the endpoint of the API backend implementation in the **Endpoint** tab as Production and Sandbox endpoints.
-
-If this endpoint is secured, there is an option for you to set the Auth type, credentials for the endpoint and endpoint certification under **General Endpoint Configurations.**
+When you [create an API](../../../../Learn/DesignAPI/CreateAPI/create-a-rest-api.md) using the API Publisher, you can specify the endpoint of the API backend implementation in the **Endpoint** tab as Production and Sandbox endpoints.
```
Comment (tgtshanika) [inline en/docs/Learn/DesignAPI/Endpoints/EndpointSecurity/basic-auth.md]: Use {{base_path}}/Learn/DesignAPI/CreateAPI/create-a-rest-api.md

File: en/docs/Learn/DesignAPI/Endpoints/EndpointSecurity/basic-auth.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 
+1. If this endpoint is secured, there is an option for you to set the Auth type, credentials for the endpoint and endpoint certification under **General Endpoint Configurations.**
 ![general-endpoint-detail](../../../../assets/img/Learn/general-endpoint-detail.png)
```
Comment (tgtshanika) [inline en/docs/Learn/DesignAPI/Endpoints/EndpointSecurity/basic-auth.md]: Use {{base_path}}

Comment (nalakas) [reply en/docs/Learn/DesignAPI/Endpoints/EndpointSecurity/basic-auth.md]: fixed in 
f9ecbc5

File: en/docs/Learn/DesignAPI/Endpoints/EndpointSecurity/basic-auth.md (hunk 1 above)
Comment (nalakas) [reply en/docs/Learn/DesignAPI/Endpoints/EndpointSecurity/basic-auth.md]: fixed in f9ecbc5

**Pull request merged**